src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminal.accessibility.contribution.ts

```diff
@@ -42,7 +42,9 @@ class AccessibleBufferContribution extends DisposableStore implements ITerminalC
 	}
 
 	xtermReady(xterm: IXtermTerminal & { raw: Terminal }): void {
-		this._accessibleBufferWidget = this._instantiationService.createInstance(AccessibleBufferWidget, xterm, this._instance.capabilities);
+		this._instance.containerReadyBarrier.wait().then(() => {
```

Comment:
It's a bit ugly to expose this, how about instead we add `ITerminalContribution.layout` and call it for all contributions within `TerminalInstance`? That seems like it would be useful for future contributions.

For this case we would only need to listen to the first layout:

```ts
layout() {
  if (!this._doneLayout) {
    this._accessibleBufferWidget = this._instantiationService.createInstance(AccessibleBufferWidget, this._instance.instanceId, xterm);
    this._doneLayout = true;
  }
}
```

Reply:
kind of confusing, deferring https://github.com/microsoft/vscode/issues/176826